tests/start_network.py

```diff
@@ -55,6 +55,7 @@ def run(args):
             network.start_and_join(args)
 
         primary, backups = network.find_nodes()
+        max_len = len(str(len(backups)))
```

Comment:
```suggestion
        max_len = max(len(str(backup.local_node_id)) for backup in backups)
```

Else this breaks when we have 3 nodes left, but they're 9, 10, and 11 (or 1, 100, and 100000000).